bears/python/PycodestyleBear.py

```diff
@@ -35,7 +35,7 @@ def create_arguments(
         :param max_line_length:
             Limit lines to this length.
         """
-        arguments = [r"--format='%(row)d %(col)d %(code)s %(text)s'"]
+        arguments = [r"--format='%(row)d %(col)d %(code)s %(text)s"]
```

Comment:
@jayvdb I removed the double quotes and replaced with single quotes, and removed the single quotes from inside. This is the convention which was followed elsewhere in other bears. Please review